src/manifolds/Sphere.jl

```diff
@@ -147,112 +214,113 @@ Represent the tangent vector `X` at point `p` from the [`Sphere`](@ref) `M` in
 an orthonormal basis by rotating the vector `X` using the rotation matrix
 $2\frac{q q^\mathrm{T}}{q^\mathrm{T} q} - I$ where $q = p + (1, 0, …, 0)$.
 """
-function get_coordinates(M::Sphere{N}, p, X, B::DefaultOrthonormalBasis) where {N}
-    if isapprox(p[1], 1)
+function get_coordinates(M::Sphere{n,ℝ}, p, X, B::DefaultOrthonormalBasis) where {n}
+    if isapprox(abs(p[1]), 1)
```

Comment:
Why did you add `abs` here? I think there real error here is in the `else` section (`xp1` should probably be `p .+ ntuple(i -> ifelse(i == 1, -1, 0), n+1)`). Similarly for `get_vector`

Ref: https://github.com/JuliaManifolds/Manifolds.jl/pull/86#discussion_r364019155

Reply:
I added the `abs` when thinking of the complex case, but forgot to remove it when leaving this to the real case. For the real case the tuple is fine.

The basis for the complex case is something I first have to think about.